hpx/runtime/threads/policies/local_priority_queue_scheduler.hpp

```diff
@@ -1256,7 +1246,7 @@ namespace hpx { namespace threads { namespace policies {
             });
 
             // check for the rest and if we are NUMA aware
-            if (numa_sensitive_ != 2 && any(first_mask & pu_mask))
+            if (has_work_stealing_numa() && any(first_mask & pu_mask))
```

Comment:
Is the old distinction of `numa_sensitive = 0/1/2` not necessary/used anymore? I don't remember what the distinction between the three states was to be honest.